### PR TITLE
Add Lighthouse branding and developer portal links to repo wherever possible

### PR DESCRIPTION
**Pull request by ryanlink**
Added "Lighthouse" per outreach style guide for consistent branding.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/general-feedback.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ name: 😡😐😀 General Feedback
 about: For general feedback on the VA API Platform
 ---
 
+## VA Lighthouse API General Feedback
```
Comment (stephaniemace) [inline .github/ISSUE_TEMPLATE/general-feedback.md]: Could we just make this?: VA Lighthouse General Feedback

File: services/vba_documents/UnderstandingDocumentProcessing.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
-This document provides some details about how documents are processed by VA and its affiliates after they are submitted via the document upload API. 
+## VA Lighthouse Benefits Intake API - Document Processing
+
+This document provides some details about how documents are processed by VA and its affiliates after they are submitted via the Benefits Intake API.  Please visit the Lighthouse [developer portal](https://developer.va.gov/explore/benefits/docs/benefits) for the latest documentation and release notes.
```
Comment (stephaniemace) [inline services/vba_documents/UnderstandingDocumentProcessing.md]: One change to add VA before Lighthouse: Please visit the VA Lighthouse [developer portal]

File: test_accounts.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Test User Accounts
+# VA Lighthouse OAuth Test User Accounts
 
 These are test accounts you can use to test OAuth APIs by logging in with them through the ID.me sandbox environment that is connected to dev-api.va.gov.  When signing in with a test account, step 2 will ask how you want to receive an authentication code and you can just click "continue" for the verification to be automatically populated in step 3.
```
Comment (stephaniemace) [inline test_accounts.md]: Should the S in step be capitalized?

File: services/vba_documents/UnderstandingDocumentProcessing.md (hunk 2 above)
Comment (ryanlink) [reply services/vba_documents/UnderstandingDocumentProcessing.md]: done!

File: test_accounts.md (hunk 3 above)
Comment (ryanlink) [reply test_accounts.md]: done!

File: .github/ISSUE_TEMPLATE/general-feedback.md (hunk 1 above)
Comment (ryanlink) [reply .github/ISSUE_TEMPLATE/general-feedback.md]: done

**Pull request merged**